cli/pkg/local/app.go

```diff
@@ -307,6 +309,18 @@ func (a *App) Serve(httpPort, grpcPort int, enableUI, openBrowser, readonly bool
 	gctx := graceful.WithCancelOnTerminate(a.Context)
 	group, ctx := errgroup.WithContext(gctx)
 
+	// Check if given gRPC port is open
+	err = scanPort(grpcPort)
+	if err != nil {
+		return fmt.Errorf("Given GRPCport: %d alreay in use", grpcPort)
+	}
+
+	// Check if given Http port is open
+	err = scanPort(httpPort)
+	if err != nil {
+		return fmt.Errorf("Given HTPPport: %d alreay in use", httpPort)
```

Comment:
Update error messages to `"grpc port %d is already in use"` (replacing `grpc` for `http` in second case)